packages/context/src/resolver.ts

```diff
@@ -184,7 +184,7 @@ export function resolveInjectedArguments(
 
   let nonInjectedIndex = 0;
   for (let ix = 0; ix < argLength; ix++) {
-    const injection = ix < injectedArgs.length ? injectedArgs[ix] : undefined;
+    let injection = ix < injectedArgs.length ? injectedArgs[ix] : undefined;
```

Comment:
I find this change suspicious, `injection` is not changed after initialization. Is it possible to revert this line please?